Packages/vcs/Lib/VTKPlots.py

```diff
@@ -87,7 +90,7 @@ def endEvent(self,obj,event):
     if self.renWin is not None:
       if self.reRender:
         self.reRender = False
-        self._lastSize = None
+        #self._lastSize = None
```

Comment:
Setting _lastSize to None made it so we hit the clear() and replot logic in configureEvent when doing things like dragging around text widgets. Removing this line appears to have done no harm, and it makes interacting with text boxes a lot more friendly. I can remove the commented out code, if you'd like.